### PR TITLE
add optional on getRegulatedSide

### PR DESCRIPTION
**Pull request by EtienneLt**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**
no

**What kind of change does this PR introduce?**
Bug fix

**What is the current behavior?**
if the regulated side is null when validating a loadflow there is a null pointer exception

### CODE REVIEW

File: java/src/main/java/com/powsybl/dataframe/loadflow/validation/Validations.java [hunk 1]
```diff
@@ -162,7 +162,13 @@ public static DataframeMapper<List<TwtValidationData>> twtsValidationMapper() {
                 .ints("low_tap_position", TwtValidationData::getLowTapPosition)
                 .ints("high_tap_position", TwtValidationData::getHighTapPosition)
                 .doubles("target_v", TwtValidationData::getTargetV)
-                .strings("regulated_side", d -> d.getRegulatedSide().name())
+                .strings("regulated_side", d -> {
+                    if (d.getRegulatedSide().isPresent()) {
```
Comment (geofjamg) [inline java/src/main/java/com/powsybl/dataframe/loadflow/validation/Validations.java]: Could by done using .map(...)

**Review by geofjamg — CHANGES_REQUESTED** [review]
A unit test showing the NPE would be welcome.

File: java/src/main/java/com/powsybl/dataframe/loadflow/validation/Validations.java (hunk 1 above)
Comment (EtienneLt) [reply java/src/main/java/com/powsybl/dataframe/loadflow/validation/Validations.java]: I dont know how to test in python I cant set a null regulated side on a ratio tap changer

Comment (EtienneLt) [reply java/src/main/java/com/powsybl/dataframe/loadflow/validation/Validations.java]: done

**Review by geofjamg — APPROVED** [review]

**Pull request merged**